go/staking/api/api.go

```diff
@@ -451,8 +489,9 @@ type GeneralAccount struct {
 // EscrowAccount is an escrow account the balance of which is subject to
 // special delegation provisions and a debonding period.
 type EscrowAccount struct {
-	Active    SharePool `json:"active"`
-	Debonding SharePool `json:"debonding"`
+	Active             SharePool          `json:"active"`
+	Debonding          SharePool          `json:"debonding"`
+	CommissionSchedule CommissionSchedule `json:"commission_schedule"`
```

Comment:
Update `SanityCheck` below to verify all commission schedules in genesis.

Reply:
ok